src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitorContext.cs

```diff
@@ -17,14 +14,20 @@ internal struct EvalVisitorContext
         public EvalVisitorContext(SymbolContext symbolContext, StackDepthCounter stackMarker)
         {
             SymbolContext = symbolContext;
-            StackDepthCounter = stackMarker;
+            StackDepthCounter = stackMarker;            
+        }
+
+        public EvalVisitorContext(SymbolContext symbolContext, EvalVisitorContext context)
```

Comment:
>, EvalVisitorContext context [](http://example.com/codeflow?start=61&length=28)

Why do we need this arg?  (is it the same as 'this' arg)?

Reply:
Can be removed, was just an optimization

Reply:
Yes, I'll name it "previousContext"